core/src/test/java/org/apache/james/core/MailAddressTest.java

```diff
@@ -102,100 +102,54 @@ void testBadMailAddressString(String mailAddress) {
             .isInstanceOf(AddressException.class);
     }
 
-    /**
-     * Test method for {@link MailAddress#MailAddress(java.lang.String, java.lang.String)}.
-     */
     @Test
-    void testMailAddressStringString() {
-        try {
-            new MailAddress("local-part", "domain");
-        } catch (AddressException e) {
-            assertThat(false).describedAs(e.getMessage()).isTrue();
-        }
-        try {
-            MailAddress a = new MailAddress("local-part", "-domain");
-            assertThat(true).describedAs(a.toString()).isFalse();
-        } catch (AddressException e) {
-            assertThat(true).isTrue();
-        }
-    }
-
-    /**
-     * Test method for {@link MailAddress#MailAddress(javax.mail.internet.InternetAddress)}.
-     */
+    void testGoodMailAddressStringString() {
```

Comment:
StringString?

Reply:
Didn't change that name... I guess they meant testing the constructor that is accepting two Strings as param, localPart and domain... But I agree we can probably find better